docs/post-v1experimental-determineversion.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+title: POST /v1experimental/determineversion
+permalink: /post-v1experimental-determineversion/
```

Comment:
```suggestion
permalink: /post-v1-determineversion/
```
What do you think of making this the URL, so it does not need to change when it goes out of the experimental stage?

Reply:
Good point!